.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3772,13 +3772,7 @@ stages:
       continueOnError: true
 
 - stage: throughput_profiler
-  condition: > 
-    and(
-      succeeded(), 
-      ne(variables['isNgenTestBuild'], 'True'),
-      eq(variables.isMainRepository, true),
-      eq(dependencies.generate_variables.outputs['generate_variables_job.generate_variables_step.IsProfilerChanged'], 'True')
-    )
+  condition: and(succeeded(), eq(variables.isMainRepository, true))
```

Comment:
This is running on every PR, not just on master, and even when there's no profiler changes 🤔

Reply:
ok so appsec and tracer should run only when it's tracer too no?
I just did the same as what's done for those job 🤔

Reply:
This is what I see on master today 
![image](https://github.com/DataDog/dd-trace-dotnet/assets/17292193/29ecad85-8012-429d-af6d-5c956f58c4c3)

Reply:
And on a PR for the profiler
![image](https://github.com/DataDog/dd-trace-dotnet/assets/17292193/fa710690-7937-45a6-bc2e-27c48b41a435)

Reply:
So either we do every PR + master for all the jobs, either we split per type of modification for all ...